cves/CVE-2017-14170.yml

```
@@ -144,10 +167,13 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    While the subsystem has FATE regression tests, there does not appear to be
+    any test files associated with the file where the vulnerability was located
+  fix: false
+  fix_answer: |
+    Fix commit only contains code changing the vulnerable code
```

```suggestion
  fix_answer:  Fix commit only contains code changing the vulnerable code
```
Looks like this line is within the 80-char limit as well.